lib/pipeline/addon/new-pipeline/route.js

```diff
@@ -22,28 +22,41 @@ let precanStages = ()=> [{
 export default Route.extend({
   globalStore: service(),
   projectStore: service('store'),
-
+  cookies: service(),
   precanStages: precanStages(),
   model: function() {
     var globalStore = this.get('globalStore');
     var projectStore = this.get('projectStore');
     let projectModel = window.l('route:application').modelFor('authenticated.project');
+    let user = window.l('service:access').me;
     let projectId = projectModel.project.id;
     let clusterId = projectModel.project.clusterId;
-    let accounts = globalStore.find('sourceCodeCredential');
+    let accounts = globalStore.find('sourceCodeCredential', null, {filter:{userId: user.id}});
```

Comment:
@gitlawr Can we change this to scope to the user?

Reply:
And, actually, here I don't need a live array because I have to pass the accounts into the `modalService` so that it can't update dynamically anyway.

Reply:
@vincent99 After discussing with @gitlawr, we may not make the `sourceCodeCredential` and `sourceCodeRepository` user-scoped. 

The reason here is that the Pipeline is shared among the same project so that User A can see the Pipeline created by User B. It means User A can access the git account information of User B. But we will control that User A can't use User B's repository to create a pipeline, or edit the pipeline created by User B. And based on what we discussed with @cloudnautique before, the User A is able to delete the Pipeline created by User B.

And, we thought that it is reasonable to let User A access User B's git account because this information is also accessible from GitHub. Nothing private here.